<launch>
  <node pkg="calib_wide_camera" type="localba" name="localba" output="screen">
    <param name="in_datapath" type="string" value="/media/sam/CR7/huawei/extrinsic/close/0/"/>
    <param name="max_iter" type="int" value="10"/>
    <param name="voxel_size" type="double" value="0.6"/>
    <param name="dsp_sz1" type="double" value="0.05"/>
    <!-- <param name="dsp_sz2" type="double" value="0.05"/> -->
    <!-- <param name="rej_ratio1" type="double" value="0.05"/> -->
    <param name="rej_ratio2" type="double" value="0.05"/>
    <!-- <param name="eigen_ratio1" type="double" value="0.05"/> -->
    <param name="eigen_ratio2" type="double" value="0.05"/>
    <param name="THR_NUM" type="int" value="16"/>
    <param name="pcd_fill_num" type="int" value="0"/><!-- use n if .pcd is started with n zeros (KITTI=5) or use 0 -->
  </node>
</launch>